frontend/src/components/shared/header/Header.tsx

```diff
@@ -1 +1,42 @@
-import React from 'react';
\ No newline at end of file
+import MailOutlineIcon from '@material-ui/icons/MailOutline';
+import SearchIcon from '@material-ui/icons/Search';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import './Header.scss';
+import logo from './logo.png';
+
+const Header: React.FC = () => {
+  const [showMenu, setMenu] = useState('');
```

Comment:
please follow naming for state [showMenu, setShowMenu]
From my point of view it is better to name this var [menuClass, setMenuClass] for this solution